code/game/objects/structures/mineral_doors.dm

```diff
@@ -27,6 +27,12 @@
 	initial_state = icon_state
 	air_update_turf(1)
 
+/obj/structure/mineral_door/Initialize()
+	. = ..()
+	initial_state = icon_state
+	air_update_turf(1)
+	AddComponent(/datum/component/rad_insulation, RAD_MEDIUM_INSULATION)

```

Comment:
make the insulation level a var and only create the component if there's insulation

Reply:
that's just needless var bloat